app/components/Form/Captcha.js

```diff
@@ -28,7 +28,7 @@ class Captcha extends Component<Props> {
           ref={ref => {
             this.captcha = ref;
           }}
-          sitekey={config.captchaKey}
+          sitekey={config.captchaKey || ''}
```

Comment:
captchaKey has a default in master now

Reply:
Right, didnt notice that it was added :)